regolith/helpers/u_milestonehelper.py

```diff
@@ -224,6 +224,13 @@ def db_updater(self):
                 if rc.finish:
                     rc.status = "finished"
                     doc.update({'end_date': now})
+                    notes = doc.get("notes", [])
+                    for i in range(len(notes)): 
+                        note = notes[i]
+                        parenthesis_index = note.find('()')
+                        if parenthesis_index != -1:
+                            notes[i] = note[:parenthesis_index+1] + 'x' + note[parenthesis_index+1:]
```

Comment:
always prefer f-strings to concatenating strings with +

Reply:
Corrected, so no longer need to concatenate.